pkgs/applications/audio/jackmix/default.nix

```
@@ -21,11 +25,11 @@ stdenv.mkDerivation {
     install -D jackmix/jackmix $out/bin/jackmix
   '';
 
-  meta = {
+  meta = with lib; {
     description = "Matrix-Mixer for the Jack-Audio-connection-Kit";
-    homepage = "http://www.arnoldarts.de/jackmix/";
-    license = stdenv.lib.licenses.gpl2;
-    maintainers = [ stdenv.lib.maintainers.kampfschlaefer ];
-    platforms = stdenv.lib.platforms.linux;
+    homepage = "https://github.com/kampfschlaefer/jackmix";
+    license = licenses.gpl2;
```

```suggestion
    license = licenses.gpl2Only;
```

I think. Please double check.